TypeScript.YAML-tmTheme

```diff
@@ -49,11 +49,11 @@ settings:
   settings: { vsclassificationtype: identifier }
 
 - scope: entity.name.tag
-  settings: { vsclassificationtype: xml literal - name }
+  settings: { vsclassificationtype: HTML Element Name }
 
 - scope: entity.other.attribute-name
-  settings: { vsclassificationtype: XML Attribute }
+  settings: { vsclassificationtype: HTML Attribute Name }
 
 - scope: meta.tag string.quoted, meta.tag string.quoted punctuation.definition.string, meta.tag string.quoted constant.character.escape
-  settings: { vsclassificationtype: xml literal - attribute value }

```

Comment:
You would also want to do this in [TypeScriptReact.Yaml-tmTheme](https://github.com/Microsoft/TypeScript-TmLanguage/blob/master/TypeScriptReact.YAML-tmTheme#L9)

Reply:
I wasn't sure what those scopes apply to, so I didn't want to risk making something worse as nobody is complaining about anything other than the attribute name.

Reply:
I disagree. https://github.com/Microsoft/TypeScript-TmLanguage/blob/master/TypeScriptReact.YAML-tmTheme#L11 has attribute value which you changed in this PR to HTML Attribute Value, you want to do same on that line. I also think we should change the  { vsclassificationtype: xml literal - text } part as well since we changed everything else to HTML. this is just incomplete change.

Reply:
Is there any way to change the color? (now JSX attrs have same color as the HTML tags and it's not good, tags are different from attributes)